drake/systems/plants/@Manipulator/parameterEstimation.m

```diff
@@ -34,17 +34,26 @@
 % @option model determines which model to use for estimation
 % 'dynamic'     = use dynamic model - requires qdd
 % 'energetic'   = use energetic model - doesn't require qdd
+% 'simerr'      = minimize simulation error
 % 
 % @option method determines which method to use for estimation
 % 'nonlinprog'    = nonlinear least squares (LS) to solve problem
 % 'linprog'       = linear LS on lumped params then nonlinear LS to recover
```

Comment:
If you say linprog, i think it's doing more than least squares. (calling a solver, etc).